packages/google_fonts/generator/generator.dart

```diff
@@ -176,25 +172,6 @@ class _FamiliesDelta {
 
     return diff;
   }
-
-  // Use cider to update CHANGELOG.md and pubspec.yaml.
-  Future<void> updateChangelogAndPubspec() async {
```

Comment:
can you log output to console to make CHANGELOG updates easier to write?

Reply:
In this repo we write changelog entries as part of the pull request.

Reply:
I understand. The list of fonts can still be output by the generator to help writing the CHANGELOG update

Reply:
Adding new code generator logic is out of scope in this PR. Maintainers can decide how they want to manage CHANGELOG updates and update the script accordingly in follow-ups.